### PR TITLE
👷 HOCS-4101 Add a KD --dryrun step

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: kube/.drone.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+steps:
+  - name: dry-run kd dev
+    image: quay.io/ukhomeofficedigital/kd:v1.19.13
+    environment:
+      DRY_RUN: true
+      VERSION: test
+      ENVIRONMENT: wcs-dev
+      KUBE_TOKEN: test
+      POISE_IPS: "127.0.0.1/32"
```
Comment (LiamMacP) [inline kube/.drone.yml:39]: Not required

File: kube/.drone.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      ACPTUNNEL_IPS: "127.0.0.1/32"
```
Comment (LiamMacP) [inline kube/.drone.yml:40]: Not required

File: kube/.drone.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      - clone
+
+  - name: dry-run kd prod
+    image: quay.io/ukhomeofficedigital/kd:v1.19.13
+    environment:
+      DRY_RUN: true
+      VERSION: test
+      ENVIRONMENT: cs-prod
+      KUBE_TOKEN: test
+      POISE_IPS: "127.0.0.1/32"
```
Comment (LiamMacP) [inline kube/.drone.yml:53]: Not required

File: kube/.drone.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      ACPTUNNEL_IPS: "127.0.0.1/32"
```
Comment (LiamMacP) [inline kube/.drone.yml:54]: Not required

File: .drone.yml [hunk 5]
```diff
@@ -250,3 +250,66 @@ steps:
       target:
         include:
           - "*-prod"
+
```
Comment (LiamMacP) [inline .drone.yml:253]: With the drone tree plugin installed - will this one be required?

File: .drone.yml [hunk 6]
```diff
@@ -250,3 +250,66 @@ steps:
...
+---
+
+kind: pipeline
+type: kubernetes
+name: lint-k8s
```
Comment (LiamMacP) [inline .drone.yml:258]: When does this get run?

**Comment by nickmcmahon01** [comment]
Close this while we look at Drone tree.

**Pull request closed without merging**